src/main/resources/assets/twilightforest/lang/es_ES.lang

```diff
@@ -0,0 +1,608 @@
+#PARSE_ESCAPES
+
+itemGroup.twilightForest=Bosque Crepuscular
+
+advancement.twilightforest.desc = Entra en los misteriosos bosques mágicos: El Bosque Crepuscular
```

Comment:
Uh, sup with these spaces before and after the =? You didn't use Yandex/Bing/Google Translate I don't think (I checked) but it's quite odd.

Reply:
Use the translator that goes directly on the google page when you search for it and then try to adjust the translation to a more human Spanish. (like now).
Spaces are a problem?

Reply:
They are a problem yes, try testing this in game, you won't get the results you expect.

Reply:
More just thought they looked ugly but that sounds like a bigger issue.